pyasic/config/mining.py

```diff
@@ -417,6 +430,10 @@ def from_epic(cls, web_conf: dict):
                 if algo_info.get("VoltageOptimizer") is not None:
                     return cls.hashrate_tuning(
                         hashrate=algo_info["VoltageOptimizer"]["Target"],
+                        throttle_limit=algo_info["VoltageOptimizer"][
```

Comment:
Maybe these should be gets, better backwards compatibility for loading if nothing else.

Reply:
Good suggestion, also allowed for a way to do backwards compat for pretty cheap.